client/client-multi/client-c/src/kaa_profile.h

```diff
@@ -59,9 +59,18 @@ typedef struct kaa_profile_manager_t kaa_profile_manager_t;
  *
  * @return      Error code.
  */
-kaa_error_t kaa_profile_update_profile(kaa_profile_manager_t *self, kaa_profile_t *profile);
+kaa_error_t kaa_profile_manager_update_profile(kaa_profile_manager_t *self, kaa_profile_t *profile);
```

Comment:
Header template should be updated as well (please, see resources of the Control server).